sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/ErrorInformation.java

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.digitaltwins.core.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+/** Error definition. */
+@Fluent
+public final class ErrorInformation {
+    /*
+     * Service specific error code which serves as the substatus for the HTTP
+     * error code.
+     */
+    @JsonProperty(value = "code", access = JsonProperty.Access.WRITE_ONLY)
```

Comment:
I'm wondering if these annotations are needed, since the validation already occurs in the impl models.

Reply:
Same thing applies to other places where, e.g., read-only attributes are set.

Reply:
Fixed